### PR TITLE
Added eslint and prettier ignore files, ran prettier and eslint on al…

### LINKED ISSUE #34: Setup staging environment

### PR DESCRIPTION
**Pull request by AlexanderPuhl**
…l files

### CODE REVIEW

**Comment by AlexanderPuhl** [comment]
I got this staging branch deployed on Netlify at this link:

https://staging--lively-lily-870c2e.netlify.app/

Gonna have this setup so we deploy to staging first, then deploy to production if all is well.

File: .eslintrc [hunk 1]
```diff
@@ -21,6 +19,7 @@
 			{
 				"argsIgnorePattern": "^_"
 			}
-		]
+		],
+		"@typescript-eslint/no-explicit-any": "off"
```
Comment (jgbowser) [inline .eslintrc:23]: I think we should consider turning this on as it is a TS best practice. I can think of one place in the code right now that has an explicit `any` and that would be the sessionize type added to the Window, are there other places that currently exist that are explicitly typed as `any`?

File: tsconfig.json [hunk 2]
```diff
@@ -10,6 +10,7 @@
 		"esModuleInterop": true,
 		"module": "esnext",
 		"moduleResolution": "node",
+		"noImplicitAny": false,
```
Comment (jgbowser) [inline tsconfig.json:13]: Similar to above, except implicit any is even more dangerous in TS. `any` has a way of polluting anything it comes into contact with and basically shutting off typescript. Were there issues with implicit any that caused you to turn this off?

**Review by jgbowser — COMMENTED** [review]
LGTM for the most part, though I'm a little concerned about allowing explicit/implicit `any` in TS code. Its generally considered ok to have those be allowed if you are converting a large codebase to TS, but in general you should prefer to disallow and add ts-ignore comments in the rare case that an `any` is required. Curious if you were running into specific issues that lead to those settings. if so, happy to look into it with you if you'd like

**Review by jgbowser — APPROVED** [review]
LGTM, concerns addressed in issue #36

**Pull request merged**